Bento/Views/BentoReusableView.swift

```diff
@@ -1,10 +1,20 @@
+/// Existential type for reusable views (cell, header, footer) in `UITableView`, `UICollectionView`, etc.
 protocol BentoReusableView: AnyObject {
-    var containedView: UIView? { get set }
+    /// The main view to which you add your reusable view’s custom content.
+    /// - Note: This is normally system-provided as `self`'s subview.
     var contentView: UIView { get }
+
+    /// `component`-generated view that will be added to `contentView`.
+    /// - Note: Subtype of this instance can conform to `ViewLifecycleAware` to allow display handling.
+    var containedView: UIView? { get set }
+
+    /// A renderer which generates `containedView`.
+    /// - Note: Underlying type of this instance can conform to `ComponentLifecycleAware` to allow display handling.
     var component: AnyRenderable? { get set }
 }
 
 extension BentoReusableView {
+    /// Updates `component` to either replace `containedView` with a new one or reuse it.
```

Comment:
This does not accurately describe the relationship. We don't update the component — the component is opaque to Bento logic. We inspect whether the view is compatible with the component, create a new view as appropriate, and finally bind the component with the view.

Reply:
f494d65 Fixed the comment with `Uses` rather than `Updates`.